server/service/src/processors/transfer/requisition/test.rs

```diff
@@ -90,41 +90,54 @@ async fn requisition_transfer() {
     let test_handle = exec_concurrent(
         test_input,
         number_of_instances,
-        |_, test_input| async move {
+        |thread_num, test_input| async move {
             let (service_provider, request_store, response_store, item1, item2) = test_input;
 
             let ctx = service_provider.basic_context().unwrap();
+            let mut tester = RequisitionTransferTester::new(
+                &request_store,
+                &response_store,
+                &item1,
+                &item2,
+                thread_num,
+            );
 
-            let mut tester =
-                RequisitionTransferTester::new(&request_store, &response_store, &item1, &item2);
-
+            log::debug!("{}: insert", thread_num);
             tester.insert_request_requisition(&ctx.connection);
             // manually trigger because inserting the requisition doesn't trigger the processor
             ctx.processors_trigger
                 .requisition_transfer
                 .try_send(())
                 .unwrap();
+            log::debug!("{}: await_events_processed", thread_num);
```

Comment:
Will all the debug logs stay~? 

Also backend tests still seem to be failing....

Reply:
The same test is failing for you?

Reply:
Or do you mean this ? https://github.com/msupply-foundation/open-msupply/issues/5029

Reply:
> Will all the debug logs stay~?

I'm suggesting Yes, why not?

Reply:
Just curious